
{'date': 'Thu Apr 30 2020 20:34:28.430 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 10
  class oi()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Apr 30 2020 20:34:41.396 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 10
  class oi():
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu Apr 30 2020 20:34:52.527 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 12
  self.SALA1.vai;
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:38:24.576 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1 = Cena(img=SALA1)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:39:11.431 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1 = Cena(img=SALA1)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:39:53.480 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1 = Cena(img="https://i.imgur.com/Z0jbt5J.png")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:40:40.432 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1 = Cena(img="SALA")
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:41:41.303 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1=Cena(img=SALA)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:41:52.503 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1=Cena(img=SALA)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:43:42.430 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 13
  self.sala1=Cena(img=SALA)
  ^
IndentationError: expected an indented block
'''},
{'date': 'Thu Apr 30 2020 20:49:18.18 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 18
    oi()
  module <module> line 15
    self.textotalita3 = Texto (self.saladeaula, "Hoje a aula vai ser sobre uma importante escritora de literatura brasileira. Me ajude a encontrar o livro")
AttributeError: 'oi' object has no attribute 'saladeaula'
'''},
{'date': 'Thu Apr 30 2020 20:50:56.387 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    oi()
  module <module> line 15
    self.quadro.entra(sala1)
NameError: name 'sala1' is not defined
'''},
{'date': 'Thu Apr 30 2020 20:51:11.727 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    oi()
  module <module> line 15
    self.quadro.entra(SALA)
  module _spy.vitollino.main line 472
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'str' object has no attribute 'nome'
'''},
{'date': 'Thu Apr 30 2020 20:55:35.203 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    oi()
  module <module> line 20
    self.textotalita3.foi = self.habilita
AttributeError: 'oi' object has no attribute 'habilita'
'''},
{'date': 'Thu Apr 30 2020 21:18:02.214 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 22
    oi()
  module <module> line 16
    self.quadro.entra(sala1)
NameError: name 'sala1' is not defined
'''},
{'date': 'Thu Apr 30 2020 21:20:49.46 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 20
    oi()
  module <module> line 17
    self.botao.entra(self.sala1)
  module _spy.vitollino.main line 472
    self.scorer.update(valor=cena.nome, move=self.xy,
AttributeError: 'Texto' object has no attribute 'nome'
'''},
{'date': 'Thu Apr 30 2020 21:22:18.881 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 21
    oi()
  module <module> line 14
    Texto(self.botao, "ols")
AttributeError: 'oi' object has no attribute 'botao'
'''},
{'date': 'Thu Apr 30 2020 21:24:03.55 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 23
    oi()
  module <module> line 19
    self.falamenino= Texto(sala1, "Muito obrigado, Senhora Neide")
NameError: name 'sala1' is not defined
'''},
{'date': 'Mon May 04 2020 16:29:34.807 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 40
  self.pasta.entra(self
                                                                                                                                     ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon May 04 2020 16:29:53.187 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    oi()
  module <module> line 36
    self.voltar1.vai=self.sala1.vai
AttributeError: 'oi' object has no attribute 'sala1'
'''},
{'date': 'Mon May 04 2020 16:30:13.512 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    oi()
  module <module> line 44
    self.sala1.vai()
AttributeError: 'oi' object has no attribute 'sala1'
'''},
{'date': 'Mon May 04 2020 16:30:59.206 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 45
    oi()
  module <module> line 44
    self.um.vai()
AttributeError: 'Sala' object has no attribute 'vai'
'''},
{'date': 'Mon May 04 2020 16:35:16.575 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 30
  self.botao.vai=self.um.leste.vai)
                                ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon May 04 2020 18:18:43.579 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 44
  self.sistema.vai=self.um.sul)
                            ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon May 04 2020 18:18:48.123 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 60
    oi()
  module <module> line 49
    self.somatico.vai
AttributeError: 'oi' object has no attribute 'somatico'
'''},
{'date': 'Mon May 04 2020 20:05:43.763 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 62
  self.aminoacido= Elemento(img= "https://i.imgur.com/YF2cXp3.png", tit "Aminoácido",
                                                                         ^
SyntaxError: invalid syntax
'''},
{'date': 'Mon May 04 2020 20:06:33.467 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 70
    oi()
  module <module> line 65
    self.falaamido= Texto(todos.leste, "Muito obrigado, Senhora Neide")
NameError: name 'todos' is not defined
'''},
{'date': 'Mon May 04 2020 20:06:44.590 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 70
    oi()
  module <module> line 65
    self.falamido= Texto(todos.leste, "Muito obrigado, Senhora Neide")
NameError: name 'todos' is not defined
'''},
{'date': 'Mon May 04 2020 20:51:27.770 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 41
  self.sistema = Elemento (img= "https://i.imgur.com/n4R7Cs6.png", x =330, y=400, y=150, w="2px"))
                                                                                   ^
SyntaxError: keyword argument repeated
'''},
{'date': 'Mon May 04 2020 20:52:05.834 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 41
  self.sistema = Elemento (img= "https://i.imgur.com/n4R7Cs6.png", x =330, y=400, y=150, w=2px)
                                                                                   ^
SyntaxError: keyword argument repeated
'''},
{'date': 'Mon May 04 2020 20:52:23.861 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 41
  self.sistema = Elemento (img= "https://i.imgur.com/n4R7Cs6.png", x =330, y=400, y=150, w=28)
                                                                                   ^
SyntaxError: keyword argument repeated
'''},
{'date': 'Mon May 04 2020 21:04:18.333 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 41
  self.sistema = Elemento (img= "https://i.imgur.com/n4R7Cs6.png", x =330, y=400, y=150, w=28)
                                                                                   ^
SyntaxError: keyword argument repeated
'''},
{'date': 'Mon May 04 2020 21:09:32.742 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 41
  self.sistema = Elemento (img= "https://i.imgur.com/n4R7Cs6.png", x =330, y=400, y=150, w=28)
                                                                                   ^
SyntaxError: keyword argument repeated
'''},
{'date': 'Wed May 06 2020 12:59:39.936 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 80
    oi()
  module <module> line 24
    self.uma= uma = Sala(o= SALA4, s= SALA5, l= SALA6, n= SALA1)
NameError: name 'SALA5' is not defined
'''},
{'date': 'Wed May 06 2020 13:21:48.989 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 81
  oestee = Elemento(FOCO, style=dict(left=530, top=500, width=80, heigth="80px"), cena=self.um.oeste, style={"opacity": 40}, vai=self.um.sul.vai)
                                                                                                           ^
SyntaxError: keyword argument repeated
'''},
{'date': 'Wed May 06 2020 13:22:08.126 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 82
  style=dict(left=530, top=500, width=80, heigth="80px"), cena=self.um.oeste, style={"opacity": 40}, vai=self.um.sul.vai)
                                                                                   ^
SyntaxError: keyword argument repeated
'''},
{'date': 'Wed May 06 2020 13:27:04.950 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 85
  self.um.norte.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed May 06 2020 13:47:19.47 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 92
    oi()
  module <module> line 33
    self.dois = dois = Sala (n=Sala1, s= SALA7, l= SALA8, o= SALA4)
NameError: name 'Sala1' is not defined
'''},
{'date': 'Wed May 06 2020 14:35:20.900 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 103
  self.um.norte.vai()
  ^
IndentationError: unexpected indent
'''},
{'date': 'Wed May 06 2020 21:24:58.405 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 107
    oi()
  module <module> line 81
    Texto(self.um.oeste, "Como nada nessa vida é fácil, você precisará de uma senha para a proxima etapa. Há 4 códigos genéticos que estão embaralhados, você terá que desembaralhar todos e descobrir o 'nosso' aminoacido secreto. FIQUE ATENTO AS NOSSAS DICAS!! ").vai()
AttributeError: 'oi' object has no attribute 'um'
'''},
{'date': 'Wed May 06 2020 21:40:24.392 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 39
  Texto(self.um.norte, "Seja bem vindo, cientista! Hoje iremos aprender um pouquinho sobre biologia com o auxilio da nossa maravilhosa tecnologia. Vamos nessa?").vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Wed May 06 2020 21:41:09.426 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 39
  Texto(self.um.norte, "Seja bem vindo, cientista! Hoje iremos aprender um pouquinho sobre biologia com o auxilio da nossa maravilhosa tecnologia. Vamos nessa?").vai()
  ^
IndentationError: expected an indented block
'''},
{'date': 'Wed May 06 2020 21:45:50.563 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 109
    oi()
  module <module> line 40
    Texto(self.um.norte, "Seja bem vindo, cientista! Hoje iremos aprender um pouquinho sobre biologia com o auxilio da nossa maravilhosa tecnologia. Vamos nessa?").vai()
AttributeError: 'oi' object has no attribute 'um'
'''},
{'date': 'Wed May 06 2020 21:48:03.798 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 39
  self.botao=Elemento(img=BOTAO,tit="ligar",
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu May 07 2020 18:50:53.527 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 61
  self.voltar1.entra(um.leste)
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu May 07 2020 18:52:51.1 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 52
  self.botao=Elemento(img=BOTAO,tit="ligar",
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu May 07 2020 18:58:26.510 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 110
    oi()
  module <module> line 30
    self.um.sul.direita.vai = self.texto_norte_um
AttributeError: 'oi' object has no attribute 'texto_norte_um'
'''},
{'date': 'Thu May 07 2020 18:59:00.450 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 110
    oi()
  module <module> line 30
    self.um.sul.direita.vai = self.texto_norte_um
AttributeError: 'oi' object has no attribute 'texto_norte_um'
'''},
{'date': 'Thu May 07 2020 19:00:06.363 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 110
    oi()
  module <module> line 30
    self.um.sul.direita.vai = self.texto_norte_um
AttributeError: 'oi' object has no attribute 'texto_norte_um'
'''},
{'date': 'Thu May 07 2020 19:00:27.73 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 110
    oi()
TypeError: __init__() missing 3 positional arguments: um,uma,dois
'''},
{'date': 'Thu May 07 2020 19:00:43.168 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 110
    oi(um, uma, dois)
NameError: name 'um' is not defined
'''},
{'date': 'Thu May 07 2020 19:00:58.239 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''Traceback (most recent call last):
  module _core.main line 160
    dialog.action(lambda *_: self.start()
  module _core.supygirls_factory line 135
    self.act(self, lambda *_: self.hide() or extra()) if self.act else None
  module _core.supygirls_factory line 306
    return self._first_response(lambda: self._executa_acao(), self.extra, self.error)
  module _core.supygirls_factory line 278
    traceback.print_exc(file=sys.stderr)
  module _core.supygirls_factory line 295
    exec(self.code, glob)  # dict(__name__="__main__"))
  module <module> line 110
    oi(self, um, uma, dois)
NameError: name 'um' is not defined
'''},
{'date': 'Thu May 07 2020 19:06:15.510 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  self.botao=Elemento(img=BOTAO,tit="ligar",
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu May 07 2020 19:11:19.430 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 53
  self.botao=Elemento(img=BOTAO,tit="ligar",
  ^
IndentationError: unexpected indent
'''},
{'date': 'Thu May 07 2020 19:11:41.254 GMt-0300 (Horário Padrão de Brasília) -X- SuPyGirls -X-',
'error': '''
 module <string> line 109
  oi()
  ^
IndentationError: expected an indented block
'''},